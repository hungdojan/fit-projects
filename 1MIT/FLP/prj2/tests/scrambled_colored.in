bog
wwg
rrw
ggo gyy rbr yow
ggo grr wbr yow
oor www oby obb
yyb
yyb
grb
