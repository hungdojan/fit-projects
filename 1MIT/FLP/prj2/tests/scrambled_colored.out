bog
wwg
rrw
ggo gyy rbr yow
ggo grr wbr yow
oor www oby obb
yyb
yyb
grb

boo
www
rrr
ggg yrw bbr yow
ggg yrw bbr yow
oow ggw bby obb
yyo
yyo
grr

www
www
rrr
ggg yrr bbb oow
ggg yrr bbb oow
oow ggg yrr bbb
yyo
yyo
yyo

www
www
rrr
ggg yrr bbb oow
ggg yrr bbb oow
ggg yrr bbb oow
ooo
yyy
yyy

www
www
www
ggg rrr bbb ooo
ggg rrr bbb ooo
ggg rrr bbb ooo
yyy
yyy
yyy
